doc/tutorials/pick_and_place_with_moveit_task_constructor/launch/pick_place_demo.launch.py

```diff
@@ -8,10 +8,8 @@ def generate_launch_description():
 
     # MTC Demo node
     pick_place_demo = Node(
-        # package="mtc_tutorial",
-        # executable="mtc_tutorial",
-        package="moveit2_tutorials",
-        executable="mtc_tutorial",
+        package="mtc_tutorial",
```

Comment:
I updated the tutorial with commands to create the mtc_tutorial package with an mtc_node executable, so I updated launch file that gets referenced in the tutorial. It will no longer allow the user to use the launch file without following the tutorial and creating the mtc_tutorial package, but it will allow them to download the launch file directly and use it alongside the tutorial. One alternative is to update the commented code in the original and add a note in the tutorial to change the commented if the user downloads it instead of writing their own

Reply:
I went with option 2